frontend/src/index.template.html

```diff
@@ -1,31 +1,31 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title><%= htmlWebpackPlugin.options.productName %></title>
-
-    <meta property="og:title" content="POAP Vote" />
-    <meta property="og:type" content="website" />
-    <meta property="og:url" content="https://poap.vote" />
-    <meta property="og:image" content="https://www.poap.xyz/POAP.f74a7300.svg" />
-    <meta property="og:image:alt" content="The POAP logo" />
-    <meta property="og:description" content="A sybil-resistant voting engine for anyone to run polls using Ethereum and POAP" />
-
-    <meta charset="utf-8">
-    <meta name="description" content="<%= htmlWebpackPlugin.options.productDescription %>">
-    <meta name="format-detection" content="telephone=no">
-    <meta name="msapplication-tap-highlight" content="no">
-    <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width<% if (htmlWebpackPlugin.options.ctx.mode.cordova || htmlWebpackPlugin.options.ctx.mode.capacitor) { %>, viewport-fit=cover<% } %>">
-
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@400;700&display=swap" rel="stylesheet">
-    <link rel="icon" type="image/png" href="statics/app-logo-128x128.png">
-    <link rel="icon" type="image/png" sizes="16x16" href="statics/icons/favicon-16x16.png">
-    <link rel="icon" type="image/png" sizes="32x32" href="statics/icons/favicon-32x32.png">
-    <link rel="icon" type="image/png" sizes="96x96" href="statics/icons/favicon-96x96.png">
-    <link rel="icon" type="image/ico" href="statics/icons/favicon.ico">
-  </head>
-  <body>
-    <!-- DO NOT touch the following DIV -->
-    <div id="q-app"></div>
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= htmlWebpackPlugin.options.productName %></title>

```

Comment:
Try to see if you can set up the variable using this syntax instead of creating the element dynamically